apps/wallet-mobile/src/NftDetails/NftDetails.tsx

```diff
@@ -137,34 +114,37 @@ const MetadataRow = ({title, copyText, children}: {title: string; children: Reac
   )
 }
 
-const NftOverview = ({nft}: {nft: Balance.TokenInfo}) => {
+// TODO: revisit (missing hit the discovery and product screen)
+const NftOverview = ({info}: {info: Portfolio.Token.Info}) => {
   const styles = useStyles()
   const strings = useStrings()
   const wallet = useSelectedWallet()
   const config = getNetworkConfigById(wallet.networkId)
 
+  const [policyId] = info.id.split('.')
+
   return (
     <View>
       <MetadataRow title={strings.nftName}>
-        <Text style={styles.name}>{nft.name}</Text>
+        <Text style={styles.name}>{info.name}</Text>
       </MetadataRow>
 
       <MetadataRow title={strings.description}>
-        <Text style={styles.name}>{normalizeMetadataString(nft.description)}</Text>
+        <Text style={styles.name}>{normalizeMetadataString(info.description)}</Text>
       </MetadataRow>
 
-      {isRecord(nft.metadatas.mintNft) && (
+      {/* {isRecord(info.metadatas.mintNft) && (
         <MetadataRow title={strings.author}>
-          <Text style={styles.name}>{normalizeMetadataString(nft.metadatas.mintNft.author)}</Text>
+          <Text style={styles.name}>{normalizeMetadataString(info.metadatas.mintNft.author)}</Text>
         </MetadataRow>
-      )}
+      )} */}
```

Comment:
cooment 👀

Reply:
yap, still ongoing changes :D checking

Reply:
aaa, it needs the tokenDiscovery endpoint, coming tmrw.